src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -686,6 +884,43 @@ def set_cont_clone_params(self,
             self.cont_clone_cmd.set_cont_clone_params(
                 dst = param)
 
+    def _set_dserial_params(self,
+                            src_pool=None, src_cont=None,
+                            dst_pool=None):
+        """Set the params for daos-serialize and daos-deserialize.
```

Comment:
@daltonbohning I'm having trouble telling if this add runs for POSIX and non-POSIX datasets? I see that you added the dataset generation for non-POSIX, but I am not familiar enough with this framework to know if it runs against both types of datasets for serialization/deserialization?

Reply:
@dsikich Everything in `data_mover_test_base.py` is "general" case. The container type is abstracted from this function. Generally, any of the tests cases that are named like `dm_posix_*` use POSIX containers. You can also look into the corresponding yaml files for `type: POSIX` to see which tests use POSIX containers. For the files touched by this PR:
`dm_obj_small.py ` - Generic container, small dataset with obj API, `dcp`
`dm_serial_small.py ` - Generic container, small dataset with obj API, serialized
Related test not touched by this PR:
`dm_obj_large_posix.py` - POSIX container, large directory, with obj API, `dcp`
Possible future test, not yet added:
`dm_serial_large_posix.py` - POSIX container, large directory, serialized

I also try to document the top of the python files with what the test is doing, as well as the test case with what is being tested. Full list of datamover tests:
https://github.com/daos-stack/daos/tree/master/src/tests/ftest/datamover

Reply:
@daltonbohning thanks, that's helpful, and answers my question about a test for posix serialization.